test/test_party_controller.py

```diff
@@ -223,28 +205,43 @@ def test_put_business_sample_link_200(self):
         self.put_to_businesses_sample_link(sample_id, put_data, 200)
 
     def test_put_business_sample_link_returns_400_when_no_ce(self):
-        mock_business = MockBusiness()\
-            .attributes(source='test_put_business_sample_link_returns_400_when_no_ce').as_business()
+        mock_business = MockBusiness().as_business()
         sample_id = mock_business['sampleSummaryId']
         self.put_to_businesses_sample_link(sample_id, {}, 400)
 
-    def test_get_business_by_ref_returns_correct_representation_verbose(self):
-        mock_business = MockBusiness() \
-            .attributes(source='test_get_business_by_ref_returns_correct_representation') \
-            .as_business()
+    def test_get_business_by_ref_returns_correct_representation(self):
+        mock_business = MockBusiness().as_business()
         self.post_to_businesses(mock_business, 200)
         self._make_business_attributes_active(mock_business)
 
-        response = self.get_business_by_ref(mock_business['sampleUnitRef'], query_string={"verbose": "true"})
-
-        del mock_business['sampleSummaryId']
-        for x in mock_business:
-            self.assertIn(x, response)
+        ru_ref = mock_business['sampleUnitRef']
+        actual = self.get_business_by_ref(ru_ref)
+        expected = {'associations': [],
```

Comment:
Is this big enough to be included as a separate JSON file (as we've done elsewhere)?

Reply:
yeah that's a fair point.  At the time I was just happy to get to work that I didn't think about tidying it up ^^